src/javascript/components/SitemapPanelApp.jsx

```diff
@@ -25,8 +26,40 @@ const SitemapPanelApp = ({client, dxContext, t}) => {
     const [sitemapIndexURL, setSitemapIndexURL] = useState(null);
     const [sitemapCacheDuration, setSitemapCacheDuration] = useState(null);
 
+    const {data, error, loading} = useQuery(gqlQueries.GetNodeSitemapInfo, {
+        variables: {
+            pathOrId: `/sites/${dxContext.siteKey}`,
+            mixinsFilter: {
+                filters: [
+                    {fieldName: 'name', value: 'jseomix:sitemap'}
+                ]
+            },
+            propertyNames: ['sitemapIndexURL', 'sitemapCacheDuration']
+        },
+        fetchPolicy: 'network-only'
+    });
+
     const [snackbarIsOpen, setSnackbarIsOpen] = useState(false);
 
+    useEffect(() => {
```

Comment:
I think we could put this one in onCompleted callback of the useQuery since we only appear to do something when we get data.

Reply:
I took a look at this for onCompleted actually
I found it doesn't really work that well with formik on how the initial value is set.